src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -144,6 +148,13 @@ def register_submission(
         context = {
             "_submission": submission,
             "productaanvraag_type": options["productaanvraag_type"],
+            "payment": {
+                "completed": submission.payment_user_has_paid,
+                "amount": submission.payments.sum_amount(),
+                "public_order_ids": ",".join(
```

Comment:
as remarked above, shouldn't this be an array of IDs instead of making the consumer comma-split it? Let's use the flexibility of JSON we have available.